frontend/src/pages/AccountSend.vue

```diff
@@ -279,12 +517,20 @@ import {
 } from 'src/utils/ethers';
 import { humanizeTokenAmount, humanizeMinSendAmount, humanizeArithmeticResult } from 'src/utils/utils';
 import { generatePaymentLink, parsePaymentLink } from 'src/utils/payment-links';
+import { SendBatch } from 'app/../umbra-js/build/src/types';
 import { Provider, TokenInfoExtended, supportedChains } from 'components/models';
 import { ERC20_ABI } from 'src/utils/constants';
 import { toAddress } from 'src/utils/address';
 
+interface SendData {
```

Comment:
Now that this component mixes both regular and batch send logic, let's try to use comments for documenting, or use more self-documenting names
```suggestion
interface BatchSendData {
```

```suggestion
// Batch send data
interface SendData {
```

Reply:
Renamed it to BatchSendData here : https://github.com/ScopeLift/umbra-protocol/pull/491/commits/7c37eca7abdc6b67f495a8154b17c3ac417361c0